File: modul_praktikum_python/contoh_code/python/rata_rata_data.py
"""Input N data dan hitung total serta rata-rata."""

# Pascal tidak menginisialisasi Total; di Python mulai dari 0.0 agar benar.


def main() -> None:
    n = int(input("Banyaknya data : "))
    total = 0.0
    for i in range(1, n + 1):
        data = float(input(f"Data ke {i} : "))
        total += data
    rata = total / n
    print("Banyaknya Data       : ", n)
    print("Total Nilai Data     : ", f"{total:5.2f}")
    print("Rata-rata nilai Data : ", f"{rata:5.2f}")


if __name__ == "__main__":
    main()

File: modul_praktikum_python/contoh_code/pascal/rata_rata_data.pas
uses crt; 
Var 
  i,N : integer;(* counter dan jumlah data *) 
  Data,Rata,Total : real;  
begin 
  write(‘Banyaknya data : ’);readln(N); 
  for i:=1 to N do 
    begin 
       write(‘Data ke ’,i,’ : ‘);readln(Data); 
       Total:=Total+Data; 
    end; 
    Rata:=Total/N; 
  writeln(‘Banyaknya Data       : ‘,N); 
  writeln(‘Total Nilai Data     : ‘,Total:5:2); 
  writeln(‘Rata-rata nilai Data : ’,Rata:5:2)  
end.
